client/src/pages/Tour.js

```diff
@@ -36,20 +36,22 @@ const Tour = () => {
   }
   return (
     <TourDiv>
-      <PageHeadline>{TourDetails.name}</PageHeadline>
-      <p>{TourDetails.description}</p>
-      <Standort>
-        {TourDetails.locationNames.map((locationNames) => (
-          <li key={locationNames}>
-            <Link to={`/location/${locationNames}`}>{locationNames}</Link>
-          </li>
-        ))}
-      </Standort>
-      <LeafletMap
-        zoomdistance="13"
-        mapCenter={["50.9375", "6.9603"]}
-        markerPosition={["50.9375", "6.9603"]}
-      />
+      {TourDetails && (
+        <>
+          <PageHeadline>{TourDetails.name}</PageHeadline>
+          <p>{TourDetails.description}</p>
+          <Standort>
```

Comment:
Why do you have the component name in German? Would it make sense to make it in English (Location)? :)

Reply:
renamed the whole component to "LocationList"